### PR TITLE
Lo-dash library update to improve js performance

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Proposed changes

Lo-dash library update to improve js performance
## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by moizjv** [comment]
Can one of the admins verify this patch?

**Comment by SrinivasanTarget** [comment]
https://github.com/appium/appium/issues/6286

File: package.json [hunk 1]
```diff
@@ -48,10 +48,10 @@
     "babel-runtime": "=5.8.24",
     "bluebird": "^2.9.25",
     "date-utils": "^1.2.18",
-    "lodash": "^3.9.1",
+    "lodash": "^4.6.1",
     "mobile-json-wire-protocol": "^1.3.2",
     "npmlog": "^2.0.1",
-    "request-promise": "^1.0.2",
+    "request-promise": "^2.0.1",
```
Comment (imurchie) [inline package.json]: I don't think we should do this until we are ready to upgrade bluebird throughout all the packages.

**Comment by imurchie** [comment]
One comment. And can you squash into a single commit?

**Comment by SrinivasanTarget** [comment]
@imurchie I squashed it now

**Comment by SrinivasanTarget** [comment]
@imurchie Help me on resolving this merge conflict. Tried my best but still it isnt resolved.

**Comment by imurchie** [comment]
Did you rebase against master?

**Comment by SrinivasanTarget** [comment]
Yes and got master is up to date

**Comment by imurchie** [comment]
Can you remove your changes to `package.json`, then re-add `winston` to it`?

**Comment by SrinivasanTarget** [comment]
I hope its resolved now

**Comment by imurchie** [comment]
That looks better.

**Comment by imurchie** [comment]
Thanks!

**Comment by SrinivasanTarget** [comment]
Wow! Thanks for your support :)

**Pull request merged**